docs/user/privacy.md

```diff
@@ -0,0 +1,61 @@
+# Privacy Policy  (Draft)
+
+## Solus Installations
+
+- Solus does not collect or store data from user's machines.
+- Solus does not use telemetry or automatic error reporting.
+- Some packages may include their own built-in telemetry (examples: Firefox, Steam). Solus cannot control data collected in this way. These packages may allow you to disable this telemetry.
+
+## Solus Websites
+
+### Dev Tracker
+
+- `dev.getsol.us`
+- You may view the dev tracker without providing personal data
+- Cookies are used to store login information
+- We collect personal data when you register for an account:
+  - Name or Username
+  - Email address
+  - Timezone (optional)
+  - SSH key (optional)
+- If you decide to contribute to packaging, your **name** and **email address** will be publicly viewable. This information will be attached to your contribution, even if you delete your account, based on our interest in maintaining documentation on submitted packages.
+- If you decide to submit bug reports or other issues, then delete your account, we will keep your submitted issues based on our interest in maintaining consistent bug reporting and issue tracking, but your name will be removed.
+
+### Forum
+
+- `discuss.getsol.us`
+- You may view the forum without providing personal data
+- Cookies are used to store login information
+- We collect personal information when you register for an account:
+  - Username
+  - Email address
+
+### Third Party Hosts
+
+Solus cannot control how data is collected and processed on the third party hosts we use. We use the following hosts:
+
+#### Cloudflare
+
+- The Solus package mirror `cdn.getsol.us` is hosted by Cloudflare
+- [Cloudflare Privacy Policy](https://www.cloudflare.com/privacypolicy/)
+
+#### Github
+
+- The Solus homepage `getsol.us`, and the help center `help.getsol.us` are hosted by Github Pages
+- Some Solus repositories are hosted on Github under the  [`getsolus` organization](https://github.com/getsolus/)
+- [Github Privacy Policy](https://docs.github.com/en/site-policy/privacy-policies/github-privacy-statement)
+
+#### Matrix
+
+- The Solus matrix rooms are hosted by the Matrix.org homeserver
+- [Matrix.org Homeserver Privacy Notice](https://matrix.org/legal/privacy-notice) 
+
+
+## Your rights under the GDPR
+
+If you are covered by the [GDPR](https://gdpr.eu/) you have rights as a data subject. Your rights are detailed under [Chapter 3 of the GDPR](https://gdpr.eu/article-12-how-controllers-should-provide-personal-data-to-the-subject/)
```

Comment:
I feel like there should be a more user-friendly link for this, but I wasn't able to quickly find one from an EU website :disappointed: